jest.config.types.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+module.exports = {
+  displayName: 'types',
+  runner: 'jest-runner-tsd',
+  testMatch: ['<rootDir>/test-types/*.test.ts'],
+};
```

Comment:
Apply this diff to get rid of some warning

```diff
diff --git c/jest.config.types.js w/jest.config.types.js
index af19558a33..727a4beee0 100644
--- c/jest.config.types.js
+++ w/jest.config.types.js
@@ -7,8 +7,30 @@
 
 'use strict';
 
+const assert = require('assert');
+
+const baseConfig = require('./jest.config');
+
+const {
+  modulePathIgnorePatterns,
+  testPathIgnorePatterns,
+  watchPathIgnorePatterns,
+} = baseConfig;
+
+assert.strictEqual(
+  testPathIgnorePatterns[0],
+  '/test-types/',
+  'First entry must be types',
+);
+
 module.exports = {
-  displayName: 'types',
+  displayName: {
+    color: 'blue',
+    name: 'types',
+  },
+  modulePathIgnorePatterns,
   runner: 'jest-runner-tsd',
   testMatch: ['<rootDir>/test-types/*.test.ts'],
+  testPathIgnorePatterns: testPathIgnorePatterns.slice(1),
+  watchPathIgnorePatterns,
 };
```

Reply:
Thanks, works like a charm now :)